README.md

```diff
@@ -119,10 +229,25 @@ Example:
 require('polyfill-service').listAllPolyfills();
 ```
 
-## cdn.polyfill.io deployment
+License
+-------
+
+The Financial Times has published this software under the [CC0 license][license].
```

Comment:
We now publish everything under MIT, so we should update this.  CC0 work can be re-licensed as MIT so the whole thing can be treated as MIT, except specific third party polyfills.  Also we still use the license so it should be present tense.  I suggest:

"Except where indicated in selected polyfill config files, the polyfill service codebase is licensed under the terms of the [MIT license].  Contributors must accept our [contribution terms].